hacking/contrib.md

```diff
@@ -21,4 +21,13 @@
 without further testing assuming they conform to style and don't do anything stupid.  The author becomes the maintainer for that target.
  - If the code is orphaned and no maintainer can be found in reasonable time, the patch author may take over as the new maintainer.
  - The current maintainer should be noted in the copyright banner at the top of the file in question.
- - The Github request review feature of pull requests should be used to request a review from the maintainer.
+ - The GitHub request review feature of pull requests should be used to request a review from the maintainer.
+
+## Checks before providing pull requests
+
+Before providing a pull request, please:
+- Check and remove trailing whitespace: `git rebase --whitespace=fix ...`
+- Test that all platforms compile: `cd src; make all_platforms`.
```

Comment:
I indeed came across this information in the hacking guide. I think it may make even more sense to point people to the [Contribution Guidelines](https://github.com/blackmagic-debug/blackmagic/blob/main/CONTRIBUTING.md), rather than putting different instructions in different places.

Now that the link to the Contribution Guidelines is included, would removing these bullet points work for you as well?

Reply:
I have removed these lines and referred only to the [Contribution Guidelines](https://github.com/blackmagic-debug/blackmagic/blob/main/CONTRIBUTING.md), to keep everything in one place.

Please let me know if this would not be acceptable.

Reply:
👍🏼  that works - adding the missing steps to the contrib guidelines (running linters, building all_platforms) would be a welcome change too in light of that. Yes, its more than acceptable. Thank you